nvme-cli.yaml

```diff
@@ -0,0 +1,61 @@
+package:
+  name: nvme-cli
+  version: 2.10.2
+  epoch: 0
+  description: NVM-Express user space tooling for Linux
+  copyright:
+    - license: GPL-2.0-only
+
+environment:
+  contents:
+    packages:
+      - bash
+      - build-base
+      - json-c
+      - libnvme
+      - libnvme
+      - libnvme-dev
+      - meson
+
+pipeline:
+  - uses: git-checkout
+    with:
+      expected-commit: eeaa08c9a0e9184f3889df0bff3d2a23db6d6294
+      repository: https://github.com/linux-nvme/nvme-cli
+      tag: v${{package.version}}
+
+  - uses: meson/configure
+    with:
+      opts: |
+        -Ddocs=man \
+        -Dudevrulesdir=/usr/lib/udev/rules.d/
+
+  - uses: meson/compile
+
+  - uses: meson/install
+
+  - runs: |
+      rm -rf ${{targets.destdir}}/usr/lib/systemd
+      rm -rf ${{targets.destdir}}/usr/lib/udev
+      rm -rf ${{targets.destdir}}/usr/share/bash-completion
+      rm -rf ${{targets.destdir}}/usr/share/zsh
```

Comment:
the package was picking up things under `/usr/lib/systemd` and `/usr/lib/dev` so removed that. 

bash and zsh completion were removed given this is a niche CLI and I was not sure if adding a completion for this would be useful here but in case if it is required happy to add that in subsequent commits.